### PR TITLE
CB-11020. Metering heartbeat versioning is broken.

### PR DESCRIPTION
**Pull request by oleewere**
See detailed description in the commit message.

it only uses version 1 or 2, not any newer one, version 3 is already used from 2020 dec 16, that means newer images uses the older metering streams instead of the new ones

### CODE REVIEW

**Comment by vaibhavgandhi12** [comment]
LGTM! 🚢 should be granted by CB team.

**Review by vaibhavgandhi12 — APPROVED** [review]
Forgot to approve in the previous comment. Please wait for final approval from CB devs.

**Comment by oleewere** [comment]
the change is straightforward as that is copied from fluent/settings.sls or telemetry/settings.sls, so merging this

**Pull request merged**
